<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_mouse_move">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>鼠标移动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_insert">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>键盘输入</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="position">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>x,y</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="insertContext">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>741</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mouse_click">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>鼠标点击</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_task_run">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>350</y>
      <width>241</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>移动到指定点后鼠标点击，键盘输入</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="curLineSpin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_insert_down">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>输入当前行并下移</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>当前输入行：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_main_ontop">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>置顶窗口</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="acSave"/>
  </widget>
  <action name="acSave">
   <property name="text">
    <string>SavePosition</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
